spec/migrations/20151026204114_add_vmware_ro_datastores_to_hosts_storages_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "spec_helper"
+require_migration
+
+describe AddVmwareRoDatastoresToHostsStorages do
+  let(:hosts_storages_stub) { migration_stub(:HostsStorage) }
+
+  migration_context :up do
+    it "Adds ID in correct region" do
+      migrate
```

Comment:
@kbrock if I do `h.reload` it complains about not having a primary key (maybe due to table rename) but what do you think of this?
```
      seq_start = ActiveRecord::Base.rails_sequence_start

      h = hosts_storages_stub.create(:host_id => 1, :storage_id => 2)
      migrate
      h = host_storages_stub.first

      expect(h.id).to eq(seq_start)
```